addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -307,25 +281,4 @@ patch(PosStore.prototype, {
     toggleEditMode() {
         this.isEditMode = !this.isEditMode;
     },
-    async updateModelsData(models_data) {
```

Comment:
recheck this (seems to be missing).

Reply:
See `initTableOrderCount` in `pos_bus.js`